snapshots/gcp/2023-12-05/global_carbon_budget.py

```diff
@@ -44,6 +52,7 @@ def main(upload: bool) -> None:
         snap = Snapshot(f"gcp/{SNAPSHOT_VERSION}/{data_file}")
 
         # Replace the full citation and description in the metadata.
+        snap.metadata.origin.attribution = ATTRIBUTION  # type: ignore
```

Comment:
(no action needed)
I was checking and I am not sure what the warning raised here actually means.

> "attribution" is not a known member of "None"Pylance[reportOptionalMemberAccess](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportOptionalMemberAccess)